Dockerfile

```diff
@@ -173,6 +173,12 @@ RUN ARCH=$(arch | sed s/aarch64/arm64/) \
     rm requirements.txt; \
     fi
 
+# Poetry 1.8.3 requires packaging version >= 22.0, see
+# https://github.com/python-poetry/poetry/issues/9029#issue-2153105325.
+# However, the above installation of ScanCode's requirements installs packaging version 21.3, see
```

Comment:
Could this also be fixed by simply swapping the `RUN` statements from lines 156 and 167 to install ScanCode before Poetry, in the hope that installing the latter will then get the `packaging` right?

Reply:
I didn't try this because it's less minimal invasive. 
This way we keep dependencies as close as possible to ScanCode's requirements.txt.

But, if you prefer the swapping solution still, I can try if it work. What do you think?

Reply:
I'm simply curious if it'd work. But actually, swapping might be a better solution as it does not hard-code `packaging==24.0.0`, which might be more future-proof with later Poetry updates.

Reply:
Ok, then let's just try here on CI.

Reply:
I've ended up building it locally...it works and ends up packaging version 24.0.